Sources/BraintreeLocalPayment/BTLocalPaymentRequest.swift

```diff
@@ -80,5 +93,75 @@ import BraintreeDataCollector
         self.phone = phone
         self.isShippingAddressRequired = isShippingAddressRequired
         self.bic = bic
+        self.intent = "sale"
+        self.returnURL = BTCoreConstants.callbackURLScheme + "://x-callback-url/braintree/local-payment/success"
+        self.cancelURL = BTCoreConstants.callbackURLScheme + "://x-callback-url/braintree/local-payment/cancel"
+    }
+    
+    enum CodingKeys: String, CodingKey {
+        case paymentType = "funding_source"
+        case amount
+        case currencyCode = "currency_iso_code"
+        case paymentTypeCountryCode = "payment_type_country_code"
+        case merchantAccountID = "merchant_account_id"
+        case email = "payer_email"
```

Comment:
After reviewing Android, I noticed we’re not sending the same parameters, such as `email`, `phone`, `givenName`, `surname`, `bic`, and `merchantAccountId`. I verified with the LPMS team the keys needed to send these parameters. It seems the gateway supports both `camel case` and `snake case`, web and Android use `camel case`.